content/en/news/releases/1.13.x/announcing-1.13.6/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.13.6
+linktitle: 1.13.6
+subtitle: Patch Release
+description: Istio 1.13.6 patch release.
+publishdate: 2022-07-25
+release: 1.13.6
+aliases:
+- /news/announcing-1.13.6
+---
+
+This release contains bug fixes to improve robustness.
+This release note describes what's different between Istio 1.13.5 and 1.13.6.
+
+FYI, [Go 1.18.4 has been released](https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE),
+which includes 9 security fixes. We recommend you to upgrade to this newer Go version.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** building router's routes orders, a `catch all` route does not short circuit other routes behind.  ([Issue #39188](https://github.com/istio/istio/issues/39188))
```

Comment:
behind **it**

also slightly confusing `router's routes orders`. Not sure what sounds better though.

Reply:
Done.